sdk/cosmosdb/cosmos/src/client/Database/Databases.ts

```diff
@@ -167,6 +169,7 @@ export class Databases {
     options?: RequestOptions
   ): Promise<DatabaseResponse> {
     if (!body || body.id === null || body.id === undefined) {
+      setDiagnostics("body parameter must be an object with an id property");
```

Comment:
Here we are directly setting the message, and not the error. Would it be better if we create the error first, set it in diagnostics, and then throw it?